lib/aruba/cucumber/hooks.rb

```diff
@@ -16,35 +16,40 @@
 end
 
 Before('@puts') do
```

Comment:
I was not sure what the `@puts`-hook is useful for, but now there can be different modes: Null -> Silent, KernelPuts and Puts.